completion/openqa-cli-completion.bash

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
```

Comment:
Replace that with:
```suggestion
#compdef openqa-cli
```
so that automatic completion works

Reply:
oops, sorry, that was meant for zsh only.
shouldn't harm in the bash file, but might be confusing :)

Reply:
I removed the shebang from the bash. I checked other files and it seems it works either way. zsh works also with compdef.